src/pages/Tenants.tsx

```diff
@@ -0,0 +1,36 @@
+import ApartmentSelector from "../components/ApartmentSelector";
+import Header from "../components/Header";
+import Navbar from "../components/Navbar";
+import DatePicker from "../components/DatePicker";
+import Searchbar from "../components/Searchbar";
+import Sidebar from "../components/Sidebar";
+import { useState } from "react";
+
+const Tenants = () => {
+  const [selectedView, setSelectedView] = useState("");
+
+  return (
+    <>
+      <div className="h-screen overflow-hidden flex flex-col">
+        <Header />
+        <Navbar />
+        <div className="flex flex-1 overflow-hidden">
+          <Sidebar setSelectedView={setSelectedView} />
```

Comment:
You  can add the ```<Header />``` ```<Navbar/>``` ```Sidebar``` in a seperate page called layout and use it in the route. And wrap the Tenants Page inside the layoutpage in the route